Makefile

```diff
@@ -1,8 +1,8 @@
-BASE_IMAGE ?= docker.io/usercont/base
+BASE_IMAGE ?= quay.io/packit/base
 SERVICE_IMAGE ?= docker.io/usercont/packit-service:dev
 WORKER_IMAGE ?= docker.io/usercont/packit-worker:dev
 WORKER_IMAGE_PROD ?= docker.io/usercont/packit-worker:prod
-TEST_IMAGE ?= packit-service-tests
+TEST_IMAGE ?= quay.io/packit/packit-service-tests
```

Comment:
isn't this the reason for failing tests? Since it tries to pull the image from the quay instead of using the local build?

Reply:
Thanks Maťo, no I don't think so.
But the ultimate goal here is to pull the image instead of building it.
It's still heavy WIP though.